### DISCUSSION TITLE
Unable to select the option on the dropdown. Android Mobile App using Appium

### DISCUSSION THREAD

**Discussion by Lakshmanan4233** [appium/appium#18811] [Q&A]
I am trying to select the option from the dropdown . But it is throwing the error .. Kindly help me to 

The Error like this 
Could not parse selector expression `new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().textContains(Testing).instance(0))`: `UiScrollable` doesn't have suitable method `scrollIntoView` with arguments [new UiSelector().textContains(Testing).instance(0)]: Could not parse selector expression `new UiSelector().textContains(Testing).instance(0)`: `UiSelector` doesn't have suitable method `textContains` with arguments [Testing]: Could not parse selector expression `new UiSelector().textContains(Testing).instance(0)`: Testing is not a string

Below this one i executed

driver.findElementByAndroidUIAutomator("new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().textContains("+optionToSelect+").instance(0))");

**Comment by jlipps** [comment]
```
 new UiSelector().textContains(Testing).instance(0)
```

The parser is correct. `Testing` is not a string. Make it a string instead (put quotes around it).

**Reply by Lakshmanan4233** [reply]
It is working . Thank you
